usaspending_api/broker/management/commands/fabs_nightly_loader.py

```diff
@@ -152,8 +152,8 @@ def handle(self, *args, **options):
         is_incremental_load = not any((reload_all, afa_ids, start_datetime, end_datetime))
 
         if is_incremental_load:
-            last_load_date = get_last_load_date()
-            logger.info("Processing data for FABS starting from %s" % last_load_date)
+            start_datetime = get_last_load_date()
+            logger.info("Processing data for FABS starting from %s" % start_datetime)
```

Comment:
personally I feel like "start_datetime" sounds like the time at which the load started running, but I'm not holding up a PR for that.

Reply:
Perhaps. But I'll leave as-is since it is a pre-existing parameter and variable